userManagement/views.py

```diff
@@ -30,3 +32,69 @@ def settings(request):
         "p_form": p_form,
     }
     return render(request, 'userManagement/settings.html', context)
+
+
+def userAccount(request):
+    if request.method == 'GET':
+        if not request.user.is_superuser:
```

Comment:
You check if they are a superuser only on get but you are not checking on Post. This can actually be entirely avoided by using @staff_member_required above the method

Reply:
use the check annotation we talked about